### PR TITLE
Migrate core/contextmenu_items.js to goog.module syntax

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from `goog_module`
- [x] My pull request is against `goog_module`
- [x] My code follows the [style guide](
      https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] My code is presented in the form suggested in the [module
      conversion guide](https://github.com/google/blockly/issues/5026)
- [x] I have run `npm test` locally already.

### Resolves

Part of #5026

### Proposed Changes

Converts `core/contextmenu_items.js` to `goog.module` syntax.

### CODE REVIEW

File: core/contextmenu_items.js [hunk 1]
```diff
@@ -115,17 +127,18 @@ Blockly.ContextMenuItems.toggleOption_ = function(shouldCollapse, topBlocks) {
...
-    preconditionFn:  function(/** @type {!Blockly.ContextMenuRegistry.Scope} */ scope) {
+    preconditionFn: function(/** @type {!ContextMenuRegistry.Scope} */
+                             scope) {
       if (scope.workspace.options.collapse) {
-        var topBlocks = scope.workspace.getTopBlocks(false);
-        for (var i = 0; i < topBlocks.length; i++) {
-          var block = topBlocks[i];
+        const topBlocks = scope.workspace.getTopBlocks(false);
+        for (let i = 0; i < topBlocks.length; i++) {
+          let block = topBlocks[i];
```
Comment (NeilFraser) [inline core/contextmenu_items.js:141]: Shouldn't this be `const`?

File: core/contextmenu_items.js [hunk 2]
```diff
@@ -85,27 +95,29 @@ Blockly.ContextMenuItems.registerCleanup = function() {
...
-Blockly.ContextMenuItems.toggleOption_ = function(shouldCollapse, topBlocks) {
-  var DELAY = 10;
-  var ms = 0;
-  for (var i = 0; i < topBlocks.length; i++) {
-    var block = topBlocks[i];
+const toggleOption_ = function(shouldCollapse, topBlocks) {
+  const DELAY = 10;
+  let ms = 0;
+  for (let i = 0; i < topBlocks.length; i++) {
+    let block = topBlocks[i];
```
Comment (NeilFraser) [inline core/contextmenu_items.js:120]: Shouldn't this be `const`?

File: core/contextmenu_items.js [hunk 3]
```diff
@@ -85,27 +95,29 @@ Blockly.ContextMenuItems.registerCleanup = function() {
...
  * @private
  */
-Blockly.ContextMenuItems.toggleOption_ = function(shouldCollapse, topBlocks) {
-  var DELAY = 10;
-  var ms = 0;
-  for (var i = 0; i < topBlocks.length; i++) {
-    var block = topBlocks[i];
+const toggleOption_ = function(shouldCollapse, topBlocks) {
+  const DELAY = 10;
+  let ms = 0;
```
Comment (NeilFraser) [inline core/contextmenu_items.js:118]: Shouldn't this be `const`?

File: core/contextmenu_items.js [hunk 4]
```diff
@@ -137,28 +150,31 @@ Blockly.ContextMenuItems.registerCollapse = function() {
...
-    preconditionFn: function(/** @type {!Blockly.ContextMenuRegistry.Scope} */ scope) {
+    preconditionFn: function(/** @type {!ContextMenuRegistry.Scope} */
+                             scope) {
       if (scope.workspace.options.collapse) {
-        var topBlocks = scope.workspace.getTopBlocks(false);
-        for (var i = 0; i < topBlocks.length; i++) {
-          var block = topBlocks[i];
+        const topBlocks = scope.workspace.getTopBlocks(false);
+        for (let i = 0; i < topBlocks.length; i++) {
+          let block = topBlocks[i];
```
Comment (NeilFraser) [inline core/contextmenu_items.js:177]: Shouldn't this be `const`?

**Review by NeilFraser — APPROVED** [review]

File: core/contextmenu_items.js (hunk 1 above)
Comment (moniika) [reply core/contextmenu_items.js:141]: No, it's value is reassigned on L146.

File: core/contextmenu_items.js (hunk 2 above)
Comment (moniika) [reply core/contextmenu_items.js:120]: No, its value is reassigned on L123.

File: core/contextmenu_items.js (hunk 4 above)
Comment (moniika) [reply core/contextmenu_items.js:177]: No, its value is reassigned on L182.

File: core/contextmenu_items.js (hunk 3 above)
Comment (moniika) [reply core/contextmenu_items.js:118]: No, its value is incremented on L124

**Comment by moniika** [comment]
Rebased to resolve conflicts.

**Pull request merged**